numba/cuda/functions.cu

```diff
@@ -0,0 +1,211 @@
+#include "cuda_fp16.h"
+
+extern "C" __device__ int
+hdiv_wrapper(
```

Comment:
I'm not sure `_wrapper` is a sufficient guard against name collisions - can we rename these so they're of the form `__numba_wrapper_{fn}` instead please?

Reply:
Done.